src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -29,7 +29,8 @@ describe("calcite-input-time-picker", () => {
   it("has defaults", async () =>
     defaults("calcite-input-time-picker", [
       { propertyName: "scale", defaultValue: "m" },
-      { propertyName: "step", defaultValue: 60 }
+      { propertyName: "step", defaultValue: 60 },
+      { propertyName: "overlayPositioning", defaultValue: "absolute" }
     ]));
 
```

Comment:
Can you wire up the `floatingUIOwner` common test helper?

Reply:
@jcfranco the `input-time-picker` internally uses the `calcite-popover` so I don't think the test is necessary and might not even work.